encrypted-media/scripts/playback-persistent-license.js

```diff
@@ -0,0 +1,104 @@
+function runTest(config, testname) {
```

Comment:
Is this test a subset of the events one above? Does it test anything additional?

It would be nice to minimize the duplication.